src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkTask.java

```diff
@@ -50,7 +50,9 @@ public class JdbcSinkTask extends SinkTask {
   public void start(final Map<String, String> props) {
     log.info("Starting JDBC Sink task");
     config = new JdbcSinkConfig(props);
+    log.info("Initializing JDBC writer");
```

Comment:
Move initialising and initialised logs inside the `initWriter` function? As it is created again in case of a retry as well.